### DISCUSSION TITLE
[OP Stack Deployment] Celestia DA ecotone upgrade requires L1 Beacon API endpoint but it shouldn't

### DISCUSSION THREAD

**Discussion by upnodedev** [ethereum-optimism/developers#343] [Deploying the OP Stack] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Using Celestia as DA but to upgrade to or beyond Ecotone, requires a beacon API endpoint. It shouldn't be required in this case as we are using Celestia as DA instead of EIP 4844 blobs.

### Logs

```
op-node-1  | t=2024-04-05T18:02:49+0000 lvl=crit msg="Application failed" message="failed to setup: unable to create the rollup node config: misconfigured L1 Beacon API endpoint: expected L1 Beacon API endpoint, but got none"
```

**Comment by sbvegan** [comment, marked as answer]
Hey @upnodedev, Celestia DA is outside of a standard OP Stack configuration, so its unlikely that we'll make modifications to the protcol to conform to third-party project. However, we have our own alt-da project in progress. So if you'd like to check that out please see [op-plasma](https://specs.optimism.io/experimental/plasma.html). I can relay this feedback to the team and see what they think.
